packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -351,6 +352,26 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
   })
   public showPresence: boolean;
 
+  /**
+   * Sets how the person-card is invoked
+   * Set to PersonCardInteraction.none to not show the card
+   *
+   * @type {PersonCardInteraction}
+   * @memberof MgtPerson
+   */
+  @property({
+    attribute: 'show-person-card',
```

Comment:
I don't like the design of this. When I read show-person-card I expect a boolean attribute.

Let's rename the attribute to person-card, this is more inline with how the mgt-person component exposes those properties.
The default value if nothing is supplied should be none, that way we're not introducing a breaking change where the person card starts showing up without explicitly asking for it.

Reply:
Thanks for updating this, I like when we have consistent and predictable behavior between our components.